stable/redis/templates/redis-slave-deployment.yaml

```diff
@@ -0,0 +1,113 @@
+{{- if .Values.cluster.enabled }}
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "redis.fullname" . }}-slave
+  labels:
+    app: {{ template "redis.fullname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+spec:
+{{- if .Values.cluster.slaveCount }}
+  replicas: {{ .Values.cluster.slaveCount }}
+{{- end }}
+  template:
```

Comment:
Am I missing the "selector.matchLabels" property?

Reply:
Not necessary in Deployment objects, which will auto-resolve the labels based on the template.

Reply:
> But this is even more important for sets like Deployment. Without this, the entire set of labels is used to select matching pods, and this will break if you use labels that change, like version or release date.

https://github.com/kubernetes/helm/blob/15053e6750ebf50e7ab28b4eecf955c53ffb5b6e/docs/chart_best_practices/pods.md#podtemplates-should-declare-selectors